app/controllers/support/responsible_bodies_controller.rb

```diff
@@ -3,15 +3,15 @@ def index
     @responsible_bodies = ResponsibleBody
       .select('responsible_bodies.*')
       .excluding_department_for_education
-      .with_users_who_have_signed_in_at_least_once
-      .with_user_count
+      .with_users_who_have_signed_in_at_least_once(privacy_notice_required: @current_user.is_computacenter?)
```

Comment:
I'm not convinced we need to filter this count... I think CC should still see that the users are there, but they just shouldn't see who they are. Otherwise it looks misleading...

Reply:
> Should we also add notes to the user lists that say "You're only seeing users who have seen the privacy policy" or something similar?

@aldavidson yes that's a good idea and the right way to go. @fofr or @ralph-hawkins could we get a bit of micro-design for this message?

Reply:
I'm with @aldavidson and think we should filter the count. I don't think CC need to know there are accounts they cannot see.